src/3rd_party/CMakeLists.txt

```diff
@@ -195,7 +195,7 @@ if ((NOT "${BSON_FOUND}") OR ("${BSON_VERSION}" VERSION_LESS "1.2.0"))
       INSTALL_DIR ${3RD_PARTY_INSTALL_PREFIX}
       DOWNLOAD_DIR ${BSON_LIB_SOURCE_DIRECTORY}
       SOURCE_DIR ${BSON_LIB_SOURCE_DIRECTORY}
-      CONFIGURE_COMMAND touch aclocal.m4 configure.ac Makefile.am Makefile.in configure config.h.in libbson.pc.in && ./configure --prefix=${3RD_PARTY_INSTALL_PREFIX}
```

Comment:
Is there a reason we have removed the libbson reference here?

Reply:
not really, seems that it has been lost during rebase to develop branch. added in f87bb9a5507d9e710839ecf9bae2fd513bab2f06